back/src/Guide/Api/Types.hs

```diff
@@ -33,7 +33,10 @@ module Guide.Api.Types
   , CSearchResult(..), toCSearchResult
 
   -- * Other types
+  , Move(..)
+  , Direction(..)
   , TraitType(..)
+  , CreateNewTrait(..)
```

Comment:
Since all of these are client types, let's keep it consistent and add prefixes: `CMove`, `CDirection`, `CTraitType`, etc. Then you can also have `CCreateTrait` instead of `CreateNewTrait`, which would be more consistent since the method is called `createTrait`.